ecf/cam/stats/jevs_href_grid2obs_stats.ecf

```diff
@@ -52,23 +52,20 @@ export COMPONENT=cam
 export RUN=atmos
 export VERIF_CASE=grid2obs
 export MODELNAME=href
-
 export KEEPDATA=YES
 
+export run_envir=dev
+
+export cyc=00
+
+export run_mpi=yes
+export gather=yes
+
+export FIXevs=/lfs/h2/emc/vpppg/noscrub/logan.dawson/CAM_verif/masks/Bukovsky_CONUS/EVS_fix
+export MASKS=/lfs/h2/emc/vpppg/noscrub/logan.dawson/CAM_verif/masks/Bukovsky_CONUS/EVS_fix
+export COMOUT=/lfs/h2/emc/vpppg/noscrub/${USER}/com/$NET/$evs_ver
```

Comment:
You would need to change your FIX and MASKS directory to the following:

export FIXevs=/lfs/h2/emc/vpppg/noscrub/emc.vpppg/verification/EVS_fix
export MASKS=/lfs/h2/emc/vpppg/noscrub/emc.vpppg/verification/EVS_fix/masks

These locations will be in a different directory when we submit the code to NCO, but for now, these are the correct mask and fix file locations.